js/Marquee.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const marqueeUrlSuffix = `/api/v3/quotes/nyse`;
+
+class Marquee {
```

Comment:
nice work here! good class!
however there is a UI issue here, the Marquee is making the width of the window very big - probably a CSS issue.